### DISCUSSION TITLE
Lidarr setup - download paths

### DISCUSSION THREAD

**Discussion by tikibozo** [RandomNinjaAtk/arr-scripts#7] [Q&A] [answered]
Moving from the -extended containers starting with Lidarr. From the scripts, it's not clear how to provide data paths for Audio.Service and Video.Service. 

For video, it looks like extended.conf is missing: "videoPath=/music-videos" (along with the additional volume in the readme), and I'm not sure where audio is supposed to end up.

It looks like the log line (Audio.Service.bash @ 135) doesn't pass a variable, and I don't think the files ever leave /config (which for me is where my docker working directory volumes live, and not where media is stored/plex libraries point.)

I could totally could misreading things with my basic bash knowledge, but I just wanted to make sure these pointed to my existing files to avoid re-downloading everything with the new setup before switching. Thx.

**Comment by RandomNinjaAtk** [comment]
Your correct about music videos, I believe it is hard coded at this time using the same path as the original container.

However, music videos doesn't work at this time because the source site is broken. And based on their track record, it may never get fixed... (Example: https://imvdb.com/n/niki)

Audio is also hard coded I believe at this point and it temporarily downloads to a sub folder in the /config/extended 

After every download the directory is wiped, so it's really only a temporary location at best.

Can look at making it a configurable option though when I have some more cycles.

**Reply by tikibozo** [reply]
I saw your note about @ mentions or something on the yt helper. fingers crossed.

Where are audio files supposed to end up? Should I set the downloadPath to something on my media volume and add the /complete folder beneath that to my Music library in Plex? For the container version, I had /music-videos and /downloads-lidarr-extended volumes. Does this refactor use the same?

**Reply by RandomNinjaAtk** [reply]
Refactor uses the same path for videos, but a new location, like I said in the last post for audio.

It should just work without you having to map anything for audio. 

It just might not be where you want the temp downloads to be. I'll make it configurable when I have some free cycles, sometime tomorrow most likely.

**Reply by tikibozo** [reply]
Oh I see - for some reason I thought it was persisting the files there (like the videos did) rather than being published into the lidarr-managed artist/release folders. I don't care that the temp directory is under /config, I just didn't want to keep media there (but that's not what's happening.) Thanks.

**Reply by RandomNinjaAtk** [reply]
Some bug fixes were just implemented, update your local config file with the latest options from here: https://github.com/RandomNinjaAtk/arr-scripts/blob/main/lidarr/extended.conf

Restart the container twice to pull the updated scripts... There is no method to auto-update the config file with new settings.

Wait for first restart to fully complete, then restart again for the new scripts to take over...

Edit: 
Update instructions: https://github.com/RandomNinjaAtk/arr-scripts/blob/main/lidarr/readme.md#updating

**Comment by RandomNinjaAtk** [comment, marked as answer]
Download paths added to the config file, please update accordingly:

Update Instructions: https://github.com/RandomNinjaAtk/arr-scripts/blob/main/lidarr/readme.md#updating
